adapters/bridge_test.go

```diff
@@ -84,11 +84,11 @@ func TestBridge_Perform_startANewRun(t *testing.T) {
 
 			bt := cltest.NewBridgeType("auctionBidding", mock.URL)
 			eb := &adapters.Bridge{BridgeType: bt}
-			input := cltest.RunResultWithValue("lot 49")
+			input := cltest.RunResultWithResult("lot 49")
```

Comment:
Yeah, this Tracker story was a bit half baked. I was also considering renaming `Data` to result, because that would make the actual result object parody the JSON-RPC response with `result` and `error`.